_data/glossary/controller.yaml

```diff
@@ -0,0 +1,12 @@
+id: controller
+name: controller
+full-link: /docs/admin/kube-controller-manager/
+related:
+- controller-manager
+tags:
+- architecture
+- fundamental
+short-description: >
+  A control loop that watches the shared state of the cluster through the {% glossary_tooltip text="apiserver" term_id="kube-apiserver" %} and makes changes attempting to move the current state towards the desired state. Examples of controllers that ship with Kubernetes today are the replication controller, endpoints controller, namespace controller, and serviceaccounts controller 
```

Comment:
Please consolidate the examples in the long description.

Reply:
Without it, [the page](https://deploy-preview-6600--kubernetes-io-master-staging.netlify.com/docs/concepts/overview/components/) will not render properly.